### ISSUE TITLE
When returning to Terms and Conditions page from "detail" view, bounce focus to "accept and continue" button

### ISSUE THREAD

**Issue by hbillings** [18F/cms-hitech-apd#1677]
When navigating from the "this is a government website" page to the "read more details" page and back, some weirdness occurs with links. Below you can see the flow that currently happens, in which a user is bounced to a nonexistant button when they come back to the auth screen after reading the details. Now you gotta tab through all the links again to get back to the "Accept" button.

Also, buttons appear to be present when they are, in fact, not. I'm not sure if this has something to do with how the DOM is refreshed or not, but this could potentially be problematic for things like the `Review` component if so.

## Step 1
<img width="1053" alt="Screen Shot 2019-07-05 at 12 38 17 PM" src="https://user-images.githubusercontent.com/509309/60740330-845e7c80-9f22-11e9-998f-b6eea2e6abbd.png">

## Step 2
<img width="771" alt="Screen Shot 2019-07-05 at 12 38 52 PM" src="https://user-images.githubusercontent.com/509309/60740333-89233080-9f22-11e9-9b67-cd663cdb236a.png">

## Step 3
<img width="1071" alt="Screen Shot 2019-07-05 at 12 39 12 PM" src="https://user-images.githubusercontent.com/509309/60740338-8cb6b780-9f22-11e9-8d48-6fe9333a525c.png">

### This task is done when...
 - [ ] Buttons that are not on the page are not clickable
 - [ ] When returning to the auth page from the detail view, "Accept and Continue" is selected by default.
 
*Add a LABEL (design, dev, compliance, BUG, etc) before submitting.

*If the issue is needed to complete prioritized work for the CURRENT SPRINT, add it to the "This Sprint" pipeline. Otherwise, all other issues will be automatically added to the _unprioritized pipeline_ for PRODUCT to prioritize.

**Comment by NAretakis** [comment]
Read more details, "pop-out in place" should be a low LOE solution.
